tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "skipLibCheck": true,
     "noImplicitAny": true,
     "esModuleInterop": true,
-    "declaration": true // enable this once all files are .ts and we can remove allowJs
+    "declaration": true, // enable this once all files are .ts and we can remove allowJs
```

Comment:
Can we nix this comment? `declaration` is already true and [`allowJs`](https://github.com/probot/probot/blob/master/tsconfig.json#L3) is already `false`. Also json doens't support comments in the first place.